recipe_scrapers/marleyspoon.py

```diff
@@ -0,0 +1,146 @@
+import re
+
+from ._abstract import AbstractScraper
+from ._exceptions import RecipeScrapersExceptions
+from ._utils import get_minutes, normalize_string
+
+
+class UnsupportedLocale(RecipeScrapersExceptions):
+    """No support for selected locale of this website"""
+
+    def __init__(self, lang):
+        self.lang = lang
+        message = f'Selected locale "{self.lang}" is not supported.'
+        super().__init__(message)
+
+
+class MarleySpoon(AbstractScraper):
+    @classmethod
+    def host(cls, domain="de"):
+        return f"marleyspoon.{domain}"
+
+    def canonical_url(self):
+        return self.soup.find("meta", {"property": "og:url"})["content"]
+
+    def author(self):
+        return normalize_string(self.soup.find(class_="author-info").a.get_text())
+
+    def title(self):
+        return normalize_string(
+            self.soup.find("h3", {"class": "recipe-title"}).get_text()
+        )
+
+    def _specs(self):
+        vocab = {
+            "de": {
+                "time": "Dauer",
+                "category": "Schwierigkeitsgrad",
+                "nutrients": "Nährwertangaben",
+            },
+            "nl": {
+                "time": "Bereidingstijd",
+                "category": "Niveau",
+                "nutrients": "Voedingswaarde",
+            },
+            "en": {
+                "time": "Serving Time",
+                "category": "Level",
+                "nutrients": "Nutrition",
+            },
+        }
+
+        self.locale = self.language()[:2]
+        if self.locale not in vocab.keys():
+            raise UnsupportedLocale(self.locale)
+
+        recipe_specs = self.soup.find("ul", {"class": "recipe-specs"})
+
+        time_label = recipe_specs.find("label", string=vocab[self.locale]["time"])
+        category_label = recipe_specs.find(
+            "label", string=vocab[self.locale]["category"]
+        )
+        nutrients_pattern = "^{}.*$".format(vocab[self.locale]["nutrients"])
+        nutrients_label = recipe_specs.find(
+            "label", string=re.compile(nutrients_pattern)
+        )
+
+        time_info = time_label.find_next_sibling("p").get_text()
+        category_info = category_label.find_next_sibling("p").get_text()
+        nutrients_info = nutrients_label.find_next_sibling("p").get_text()
+
+        return time_info, category_info, nutrients_info
+
+    def total_time(self):
+        return get_minutes(self._specs()[0])
+
+    def category(self):
+        return normalize_string(self._specs()[1])
+
+    def nutrients(self):
+        nutrients = {}
+
+        vocab = {
+            "calories": {
+                "de": "Kalorien",
+                "nl": "Calorieën",
+                "en": "Calories",
+                "unit": "calories",
+            },
```

Comment:
Thank you for introducing these matching terms :+1: 

I wondered a little bit about the fact that we're parsing units in multiple languages yet arguably losing some information by normalizing those to a single output natural language.

This is, however, already true of the names of the output keys too (`carbohydrateContent` for example), so it is consistent - and more importantly it [corresponds with an existing defined schema](https://www.schema.org/NutritionInformation).

So I think all is well here - thanks again!

Reply:
Good point. I guess the strings would have to be translated back by apps using the scrapers.